### PR TITLE
Phoenix (constellation)

### PR DESCRIPTION
**Pull request by flyingbarron**
https://github.com/instructlab/taxonomy/issues/830

If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- ...
- ...
- ...

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

File: knowledge/astronomy/constellations/Phoenix/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+created_by:
```
Comment (jjasghar) [inline knowledge/astronomy/constellations/Phoenix/qna.yaml:1]: Please put your GitHub Username here.

File: knowledge/astronomy/constellations/Phoenix/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+created_by: 
+domain: constellation
+seed_examples:
+- question: When was the constellation Phoenix defined? 
+  answer: |
+	  It first appeared on a 35-cm diameter celestial globe published in 1597 (or 1598) in Amsterdam by Petrus Plancius with Jodocus Hondius. It was first depicted on a celestial atlas by Johann Bayer in his 1603 Uranometria. The French explorer and astronomer Nicolas Louis de Lacaille charted the brighter stars and gave their Bayer designations in 1756.
```
Comment (jjasghar) [inline knowledge/astronomy/constellations/Phoenix/qna.yaml:6]: Tab is bad. :)

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the created_by, and the tab. Then we can run the linter.

File: knowledge/astronomy/constellations/Phoenix/qna.yaml (hunk 2 above)
Comment (flyingbarron) [reply knowledge/astronomy/constellations/Phoenix/qna.yaml]: done

File: knowledge/astronomy/constellations/Phoenix/qna.yaml (hunk 1 above)
Comment (flyingbarron) [reply knowledge/astronomy/constellations/Phoenix/qna.yaml]: done

**Review by flyingbarron — COMMENTED** [review]
Made changes

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the linting issues

File: knowledge/astronomy/constellations/Phoenix/qna.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+created_by: flyingbarron
+domain: constellation
+seed_examples:
+- question: When was the constellation Phoenix defined?
+  answer: | It first appeared on a 35-cm diameter celestial globe published in 1597 (or 1598) in Amsterdam by Petrus Plancius with Jodocus Hondius. It was first depicted on a celestial atlas by Johann Bayer in his 1603 Uranometria. The French explorer and astronomer Nicolas Louis de Lacaille charted the brighter stars and gave their Bayer designations in 1756.
```
Comment (jjasghar) [inline knowledge/astronomy/constellations/Phoenix/qna.yaml:5]: Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the linting issue.

File: knowledge/astronomy/constellations/Phoenix/qna.yaml (hunk 3 above)
Comment (flyingbarron) [reply knowledge/astronomy/constellations/Phoenix/qna.yaml]: Done, I think.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please resolve the linting issues.

**Comment by jjasghar** [comment]
@instructlab-bot precheck

**Comment by jjasghar** [comment]
Closing per the combined PR here: https://github.com/instructlab/taxonomy/pull/1159

**Pull request closed without merging**